ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -310,9 +310,9 @@
 
   public static final String KSQL_SUPPRESS_BUFFER_SIZE = "ksql.suppress.buffer.size";
   public static final Long KSQL_SUPPRESS_BUFFER_SIZE_DEFAULT = -1L;
-  public static final String KSQL_SUPPRESS_BUFFER_SIZE_DOC = "Bound the size of the buffer used "
-      + "for suppression. Negative size means the buffer will be unbounded. If the buffer exceeds "
-      + "its max capacity, a StreamsException stating this is thrown";
+  public static final String KSQL_SUPPRESS_BUFFER_SIZE_DOC = "Bound the number of bytes that the "
```

Comment:
perhaps we should even go so far as to change the config to be `ksql.suppress.buffer.size.bytes` - thoughts?

Reply:
I think that's a good idea in order to avoid any ambiguity